### ISSUE TITLE
Appendix B - ONRR Lease Conversions

### ISSUE THREAD

**Issue by merrellc** [ONRR/onrr.gov-site#1773]
Hello, please replace the PDF for Appendix B on the website with the attached document. 

Document is located at -
https://onrr.gov/reportpay/handbooks/revenue-handbook/index.htm

Thank you
[Appendix B ONRR Lease, Unit, or Communitization Agreement Number..pdf](https://github.com/ONRR/onrr.gov-site/files/8375345/Appendix.B.ONRR.Lease.Unit.or.Communitization.Agreement.Number.pdf)

**Comment by cthomasONRR** [comment]
@merrellc Good Morning,
Appendix B of the MRRH is updated on the dev site: https://www.onrr.gov/reportpay/Handbooks/revenue-handbook/index.htm.
Please review and give us your approval to go live.

@Maroyafaied this document is also now tagged with 'update' in the CMS.

Thanks,
Christine

**Comment by cthomasONRR** [comment]
@merrellc Yes, that's where I updated the document. http://2017onrr/reportpay/Handbooks/revenue-handbook/index.htm

**Comment by merrellc** [comment]
Must not have been updated when I originally looked.

Thank you! Document is ready to go live.


Kind Regards,
​Candace Merrell
Minerals Revenue Specialist - Staff
Data Solutioning and Technical Support
Reference and Reporting Management
Office of Natural Resources Revenue
303-231-3517
***@***.***

Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.
________________________________
From: Christine Thomas ***@***.***>
Sent: Wednesday, March 30, 2022 8:47 AM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Merrell, Candace J ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Appendix B - ONRR Lease Conversions (Issue #1773)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@merrellc<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmerrellc&data=04%7C01%7Ccandace.merrell%40onrr.gov%7C7d709d172d4c42b03f6b08da125c554d%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637842485023656709%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000&sdata=Gc%2B6LIRkRRa5GYWGBOUjYKvPJIqzj8XFEzrrf4NLTfU%3D&reserved=0> Yes, that's where I updated the document. http://2017onrr/reportpay/Handbooks/revenue-handbook/index.htm

**Comment by cthomasONRR** [comment]
@merrellc Great, the changes are now live: https://www.onrr.gov/reportpay/Handbooks/revenue-handbook/index.htm. Please remember to use 'Ctrl+F5' to refresh the page.

Closing this issue now.

Thanks,
Christine

**Comment by mcharg** [comment]
File updated in CMS.
